plasmapy/analysis/tests/test_nullpoint.py

```diff
@@ -0,0 +1,269 @@
+"""
+Tests for the null point finder class defined in `plasmapy.analysis.nullpoint`.
+"""
+import numpy as np
+import pytest
+
+from plasmapy.analysis.nullpoint import (
+    ATOL,
+    bilinear_root,
+    jacobian,
+    locate_null_point,
+    nullpoint_find,
+    reduction,
+    trilinear_approx,
+    trilinear_coeff_cal,
+    trillinear_analysis,
+    vector_space,
+)
+
+
+def vspace_func_1(x, y, z):
+    return [(y - 5.5), (z - 5.5), (x - 5.5)]
+
+
+def vspace_func_2(x, y, z):
+    return [2 * y - z - 5.5, 3 * x + z - 22, x ** 2 - 11 * x + y + 24.75]
+
+
+def test_trilinear_coeff_cal():
+    vspace1_args = {
+        "x_range": [0, 10],
+        "y_range": [0, 10],
+        "z_range": [0, 10],
+        "precision": [10 / 46, 10 / 46, 10 / 46],
+        "func": vspace_func_1,
+    }
+    vspace1 = vector_space(**vspace1_args)
+    vspace2_args = {
+        "x_range": [0, 10],
+        "y_range": [0, 10],
+        "z_range": [0, 10],
+        "precision": [10 / 46, 10 / 46, 10 / 46],
+        "func": vspace_func_2,
+    }
+    vspace2 = vector_space(**vspace2_args)
+    test_trilinear_coeff_cal_values = [
+        (
+            {"vspace": vspace2, "cell": [25, 25, 25]},
+            [
+                [-5.5, 0, 2, -1, 0, 0, 0, 0],
+                [-22, 3, 0, 1, 0, 0, 0, 0],
+                [-5.96833648, 0.08695652, 1, 0, 0, 0, 0],
+            ],
+        )
+    ]
+    test_trilinear_coeff_cal_expections = []
+
+    test_trilinear_coeff_cal_warnings = []
```

Comment:
Were there still tests that we wanted to add about exceptions and warnings?  If not, we could delete these, at least for the time being.